Description: Poke requested Debian buildflags for hardening into Makefile
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Last-Update: 2015-01-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/Makefile
===================================================================
--- trunk.orig/Makefile
+++ trunk/Makefile
@@ -402,8 +402,8 @@ PLUGIN_CFLAGS = -O2 -g -pipe
 endif
 endif
 
-CFLAGS += $(PROFILE_FLAGS) $(REQUIRED_CFLAGS)
-PLUGIN_CFLAGS += $(PROFILE_FLAGS) $(REQUIRED_CFLAGS)
+CFLAGS += $(PROFILE_FLAGS) $(REQUIRED_CFLAGS) -fstack-protector-strong
+PLUGIN_CFLAGS += $(PROFILE_FLAGS) $(REQUIRED_CFLAGS) -fPIC -fno-stack-protector
 
 # Required for gudev-1.0
 CFLAGS += -DG_UDEV_API_IS_SUBJECT_TO_CHANGE
